### ISSUE TITLE
obvious crash on editing outputs

### ISSUE THREAD

**Issue by hurzl** [abarisain/dmix#790]
```
           java.lang.UnsupportedOperationException: Illegal argument caused by mutation attempt
  at com.anpmech.mpd.commandresponse.ObjectResponse.addAll(ObjectResponse.java:123)
  at com.namelessdev.mpdroid.fragments.OutputsFragment$UpdateAudioOutputs.run(OutputsFragment.java:323)
```

**Comment by avuton** [comment]
Thanks for the report, fix should be there in 24hr so.

**Comment by avuton** [comment]
Fixed via 43fb0f9.
